Fixes overmap events not generating properly

comma opened the pull request:
They couldn't do it because proc they were using to get turfs around them was light sensitive.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

PsiOmegaDelta commented on code/game/turfs/turf.dm:
```diff
@@ -179,7 +179,7 @@ var/const/enterloopsanity = 100
 /turf/proc/AdjacentTurfs(var/check_blockage = TRUE)
 	. = list()
-	for(var/turf/t in oview(src,1))
+	for(var/turf/t in trange(1,src))
```
Doesn't trange() include the src-turf as well?

PsiOmegaDelta replied:
This would be a change of behavior, if so.

comma committed:
Fixes overmap events not generating properly

They couldn't do it because proc they were using to get turfs around them was light sensitive.
